### DISCUSSION TITLE
What's the process of Calibrating iRT Values to PredictedRT in DIA-NN?

### DISCUSSION THREAD

**Discussion by ec-ho-ra-mos** [vdemichev/DiaNN#833] [General]
In the DIA-NN paper, a linear retention time predictor was mentioned. I believe this predictor outputs dimensionless iRT values. Could you explain how DIA-NN transforms these iRT values into PredictedRT values?

**Comment by vdemichev** [comment]
Any two RT scales can be mapped to each other, this is what happens during alignment of an arbitrary scale in the library to the empirically measured RTs in the MS run. The predictor used in DIA-NN now is based on deep learning, the legacy one described in the paper is no longer recommended.

**Reply by ec-ho-ra-mos** [reply]
Thank you very much for your reply! I am interested in learning whether DIA-NN employs an approach similar to MaxQuant’s first search/main search for RT alignment.

For deep learning-based retention time (RT) predictor algorithms, a calibration file is typically required, as seen in the case of DeepLC. I am curious about how DIA-NN generates an equivalent file/information for iRT calibration.
